feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle-dropdown/configurator-attribute-single-selection-bundle-dropdown.component.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { ConfigFormUpdateEvent } from '../../../form/configurator-form.event';
+import { Configurator } from '../../../../core/model/configurator.model';
```

Comment:
is this import correct? I mean, isn't it importing from outside the lib? please check

Reply:
there are imports to product-configurator/rulebased/core/model files so are in same lib